examples/complete/fixtures.defaults.tfvars

```diff
@@ -0,0 +1,11 @@
+region = "us-east-2"
+
+availability_zones = ["us-east-2a", "us-east-2b"]
+
+namespace = "eg"
+stage     = "test"
+name      = "vpc-adopts"
+
+default_security_group_deny_all = true
+default_route_table_no_routes   = true
+default_network_acl_deny_all    = true
```

Comment:
These inputs will need to be added to `fixtures.us-east-2.tfvars` with `false` values